q2_amr/card/mags.py

```diff
@@ -46,10 +46,7 @@ def annotate_mags_card(
             shutil.move(f"{tmp}/output.json", json_path)
             samp_bin_name = os.path.join(samp_bin[0], samp_bin[1])
             frequency_df = read_in_txt(
-                path=txt_path,
-                samp_bin_name=samp_bin_name,
-                data_type="mags",
-                colname="Best_Hit_ARO",
+                path=txt_path, samp_bin_name=samp_bin_name, data_type="mags"
```

Comment:
Just to make sure: what happened to the `colname` argument? I'm guessing that was something that got reverted too? There are quite a few changes around that below too - is that something you want or should this specific change be kept?

Reply:
So that changed because these where utils that worked for different output formats that included some from RGI and they are not needed here so thats why i removed everything that was needed for the RGI formats.

Reply:
Gotcha, I guess in that case it's fine 🙌